### DISCUSSION TITLE
Github Education verify problems

### DISCUSSION THREAD

**Discussion by BoJIwEbHuK7** [community/community#86516] [GitHub Education]
The last time I had github student privileges that expired was June 23, 2023. After obtaining new student documents and filling out the information, I applied for privileges on January 8 and  January 9 2024, respectively. Each was rejected because "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jan 9, 2024 for @HeenLight." Since I was not provided an email from the college, I am using the certificate. All information in the payment section was filled out correctly(no payment method added)
The student certificate was of the same type as last year the only difference is the dates of study which are currently correct(2023/2024), initially I uploaded a photo but after rejection I took a photo on the website and still no output. Hope for a possible solution to the problem.

**Comment by BoJIwEbHuK7** [comment]
Also I don't understand how I should upload more than one document, if when I try to add more files it's only upload latest selected.

**Comment by faridnec** [comment]
Hey, the best option is to take a photo directly on the camera with your school ID. Worked for me they accepted my request in only like three days. Previously I had the same issues when I tried to upload documents from the local. I hope this also works for you

**Reply by BoJIwEbHuK7** [reply]
I tried this also, I made photo of school document, and then uploaded it - not worked.
Made photo directly on site - not worked also.

**Reply by faridnec** [reply]
I am sorry for that. What did the session say? Is it just loading or are there any further details? The last resort might be directly contacting GitHub Support. I only can try to help based on the context you have provided.

**Reply by BoJIwEbHuK7** [reply]
First attempts(maybe 3 or 4) it's uploaded normally but all was rejected because for any reason system don't like my school ID.
After that it's started sticking on "Processing image" or something like that.

**Reply by faridnec** [reply]
I got it. It was similar with my case but after that I just leave it and then the following two days they accepted me. Maybe it's about time. 

Meanwhile I suggest you to contact them via email [Github Edu Support](https://support.github.com/features/education)

**Reply by Abdirashid-dv** [reply]
I see this issue with my friend but later we talk to the support team.
I suggest you to contact Github Education Support via email [Github Edu Support](https://support.github.com/features/education)
